terraform/cross-region-infra/providers.tf

```diff
@@ -0,0 +1,26 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+}
+
+
+provider "aws" {
+  region = var.aws_region
+}
+
+# Provider can have only static alias so I hardcoded region value here.
+# https://github.com/hashicorp/terraform/issues/9448
+provider "aws" {
+  alias = "eu-central-1"
+  region = "eu-central-1"
+}
```

Comment:
(Question) Similar issue here. I was hoping to pass two input variables (primary and secondary region) from shell script. However, the `alias` field only accepts hard-coded value.